prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -1047,7 +1047,10 @@ void rxr_pkt_handle_recv_completion(struct rxr_ep *ep,
 			 * Application called fi_av_remove() to remove the address
 			 * from address vector. In this case, this packet should be ignored.
 			 */
-			FI_WARN(&rxr_prov, FI_LOG_CQ, "Warning: ignoring a received packet from a removed address\n");
+			FI_WARN(&rxr_prov, FI_LOG_CQ,
+				"Warning: ignoring a received packet from a removed address. packet type: %" PRIu8
+				", packet flags: %" PRIu16
+				"\n", rxr_get_base_hdr(pkt_entry->pkt)->type, rxr_get_base_hdr(pkt_entry->pkt)->flags);
```

Comment:
For the flags, it is better to display it in hex format (than in decimal)

Reply:
Thanks! Fixed in the latest revision.